nx-X11/include/Imakefile

```diff
@@ -16,11 +16,8 @@ GLXDIR = GL
 
 SUBDIRS = extensions $(GLXDIR)
 
-AU_HEADERS = Xauth.h
```

Comment:
On  Mo 18 Jul 2016 15:47:45 CEST, Ulrich Sibiller wrote:

>> @@ -16,11 +16,8 @@ GLXDIR = GL
>>
>>  SUBDIRS = extensions $(GLXDIR)
>>
>> -AU_HEADERS = Xauth.h
>
> Well, I have not blindly removed files, I have checked every single  
> file and there was no difference that could harm. I just was not  
> aware that Xauth comes from Xlib now.

No problem, simply skipping that commit should be fine.

-- 

DAS-NETZWERKTEAM
mike gabriel, herweg 7, 24357 fleckeby
mobile: +49 (1520) 1976 148
landline: +49 (4354) 8390 139

GnuPG Fingerprint: 9BFB AEE8 6C0A A5FF BF22  0782 9AF4 6B30 2577 1B31
mail: mike.gabriel@das-netzwerkteam.de, http://das-netzwerkteam.de